en-US/ranking/criteria.md

```diff
@@ -2,117 +2,218 @@
-     * Jumps can be used more freely but sparingly, as players at this level may still have trouble with them.
-     * Scroll velocity changes must not be used at this difficulty level for any other purposes than scroll velocity normalization.
+### Rules
 
-* **Normal**
-     * Maps at this level should typically start branching into patterns that don't undermap the song as heavily compared to easy and beginner.
-     * 1/2 rhythms with more complicated note placements and jumps are suitable for this difficulty.
+- **Every map in the mapset must have a background image.** Using different background images for different maps is acceptable. A mapset does not require a banner image.
+- **The resolution of a background image must be at least 1280x720.**
+- **The file size of a background image must not exceed 4 MB.**
+- **The resolution of a banner image must be at least 421x82, if included.** This resolution also represents the default size and aspect ratio.
+- **The background must be appropriate for players of 13+ years old and should not contain explicit content.**
 
-* **Hard**
-     * At this level, your creativity as a mapper is able to shine as there are far less restrictions than the previous difficulties.
-     * Maps at this difficulty can branch into light 1/4th streams with jumps interspersed in them depending on the BPM of the song.
-     * Usage of more long notes are suited for this difficulty, as players will typically be learning to play them in more frequent circumstances.
+### Guidelines
 
-* **Insane**
-     * Players at this level will be able to handle a wide variety of patterns at lower densities including but not limited to:
-          * Light Jacks
-          * Complex Long Note Usage
-          * Light Jumpstreams
-     * Denser patterns are suitable for this level if they are straightforward, and not difficult to execute - such as jumptrills.
+- The background should show relevance to the song in some way. While mappers are allowed creative freedom when it comes to choosing backgrounds, completely unrelated background images that do not compliment the song in any way should generally not be used.
 
-* **Expert**
-     * From this point on, there aren't any restrictions on how hard it should be. Feel free to be creative.
+## Audio
 
-##### Difficulty Spread Rules
+### Rules
+
+- **The mapset must contain only one audio file.**  Mapsets with multiple audio files are not eligible to be ranked.
+- **MP3 is the only allowed audio file format.**
+- **The maximum bitrate allowed for MP3 files is 192kbps.**
+- **The audio must be appropriate for players of all ages and should not include content that is offensive just for the sake of being offensive.**
+
+# Maps
+
+## Rules
+
+The following requirements are for the individual maps themselves.
+
+- **Autoplay must be able to achieve a 100% (X-Grade) score.**
+- **The maps must not have overlapping notes.**
+- **The maps cannot have more than 30 seconds of consecutive break time.**
```

Comment:
It seems a majority of us are fine with this being a guideline instead, something like "Avoid having long breaks in your map"